context/app/static/js/components/entity-search/Search/Search.jsx

```diff
@@ -39,10 +39,12 @@ function Search() {
         <Results results={results} allResultsUUIDs={allResultsUUIDs} />
       </Flex>
       <SuccessSnackbar
+        // Line 43: This key forces a re-render when the message changes and resets the timer for autoHideDuration
+        key={messages[0]}
         open={snackbarOpen}
         onClose={closeSnackbar}
-        message={message}
-        autoHideDuration={2000}
+        message={messages[0]}
+        autoHideDuration={3000}
```

Comment:
This might be a bit too brief for accessibility - screen readers and similar tools are unfortunately not quite as fast as our eyes, so they'd start reading the notification then have it disappear halfway through. Would it be possible to set this higher, e.g. 5 or 10 seconds?

cc @tsliaw

Reply:
I agree, 3 seconds is a bit short. I noted in the handoff doc that the default timer should be 6 seconds for all snackbars. If future snackbars need to be longer or we get feedback saying otherwise, we can adjust the timing then to be longer.

Reply:
The updated version uses 6 seconds as the default timer and can also be closed early if desired:

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/f1c00c2e-0508-44b8-80d4-4012542903c5)